cli/testdataseeder.php

```diff
@@ -63,6 +63,11 @@ function insert_row($table, $rowcsv) {
     $strarr = explode(",", $rowcsv);
     $type = logstore_xapi_get_type_from_table($table);
 
+    if ($table == XAPI_IMPORT_TYPE_HISTORIC) {
+        $table = XAPI_REPORT_SOURCE_LOG;
+        $type = XAPI_IMPORT_TYPE_HISTORIC;
+    }
+
```

Comment:
If you send the table as `XAPI_REPORT_SOURCE_HISTORICAL` (as in the other comment) there is no need for this at all
I added the `logstore_xapi_get_type_from_table` function which should return `XAPI_IMPORT_TYPE_HISTORIC` when supplying `XAPI_REPORT_SOURCE_HISTORICAL` as the `$table`

Reply:
This table name swap was just to avoid further changes, but I have corrected this to use a table name rather than type constant in the caller.

Reply:
I think this is still incorrect @mlynn-lp 
The intention is to add rows to `mdl_logstore_standard_log` to allow QA to select historical entries to be sent on to `mdl_logstore_xapi_log`. As it stands, this will write to `mdl_logstore_xapi_log`